auto_round/script/mllm.py

```diff
@@ -239,8 +245,13 @@ def tune(args):
         cls = MllamaForConditionalGeneration
     else:
         cls = AutoModelForCausalLM
-    model = cls.from_pretrained(
-        model_name, trust_remote_code=not args.disable_trust_remote_code, torch_dtype=torch_dtype)
+    if use_auto_mapping:
+        model = cls.from_pretrained(
+            model_name, trust_remote_code=not args.disable_trust_remote_code, torch_dtype=torch_dtype)
+    else:
+        model = cls.from_pretrained(
+            model_name, trust_remote_code=not args.disable_trust_remote_code, torch_dtype=torch_dtype,
+            device_map="auto")
```

Comment:
how about
model = cls.from_pretrained(
            model_name, trust_remote_code=not args.disable_trust_remote_code, torch_dtype=torch_dtype,
            device_map="auto" if use_auto_mapping else None)